addons/mrp/models/mrp_bom.py

```diff
@@ -587,11 +587,9 @@ def _get_default_product_uom_id(self):
     product_qty = fields.Float(
         'Quantity', default=1.0,
         digits='Product Unit', required=True)
-    allowed_uom_ids = fields.Many2many('uom.uom', compute='_compute_allowed_uom_ids')
```

Comment:
You can't delete field in stable. Just keep everything and set to empty in the compute. Remove in master